pkg/cmd/chat/utils/fileupload.go

```diff
@@ -8,19 +8,43 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func UploadFile(c *stream.Client, cmd *cobra.Command, chType, chId, userID, filePath string) (string, error) {
+type uploadType string
+
+const (
+	uploadTypeFile  uploadType = "file"
+	uploadTypeImage uploadType = "image"
+)
+
+func UploadFile(c *stream.Client, cmd *cobra.Command, chType, chId, userID, filePath, contentType string) (string, error) {
+	return uploadFile(c, cmd, uploadTypeFile, chType, chId, userID, filePath, contentType)
+}
+
+func UploadImage(c *stream.Client, cmd *cobra.Command, chType, chId, userID, filePath, contentType string) (string, error) {
+	return uploadFile(c, cmd, uploadTypeImage, chType, chId, userID, filePath, contentType)
+}
+
+func uploadFile(c *stream.Client, cmd *cobra.Command, uploadtype uploadType, chType, chId, userID, filePath, contentType string) (string, error) {
 	file, err := os.Open(filePath)
 	if err != nil {
 		return "", err
 	}
 	defer file.Close()
 
 	req := stream.SendFileRequest{
-		User:     &stream.User{ID: userID},
-		FileName: filepath.Base(file.Name()),
-		Reader:   file,
+		User:        &stream.User{ID: userID},
+		FileName:    filepath.Base(file.Name()),
+		ContentType: contentType,
```

Comment:
this isn't needed actually since backend does auto detection, will be better to drop flags for simplification